GitHubRepositories.ps1

```diff
@@ -2944,6 +2945,593 @@ filter Set-GitHubRepositoryActionsPermission
+          Recommended for contributors who triage a repository. Only applies to repositories owned
+          by organizations.
+        If no permission is specified, the team's permission attribute will be used to determine
+        what permission to grant the team on this repository.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .INPUTS
+        GitHub.Branch
+        GitHub.Content
+        GitHub.Event
+        GitHub.Issue
+        GitHub.IssueComment
+        GitHub.Label
+        GitHub.Milestone
+        GitHub.Organization
+        GitHub.PullRequest
+        GitHub.Project
+        GitHub.ProjectCard
+        GitHub.ProjectColumn
+        GitHub.Reaction
+        GitHub.Release
+        GitHub.ReleaseAsset
+        GitHub.Repository
+        GitHub.Team
+
+    .EXAMPLE
+        Set-GitHubRepositoryTeamPermission -OrganizationName microsoft -Uri https://github.com/microsoft/PowerShellForGitHub -TeamName Devs -Permission Push
+
+        Sets the Push permission for the Devs team on the microsoft/PowerShellForGitHub repository.
+
+    .EXAMPLE
+        Set-GitHubRepositoryTeamPermission -OrganizationName microsoft -OwnerName microsoft -RepositoryName PowerShellForGitHub -TeamName Admins -Permission Admin
+
+        Sets the Admin permission for the Admin team on the microsoft/PowerShellForGitHub repository.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess,
+        DefaultParameterSetName = 'TeamNameElements')]
+    param(
+        [Parameter(ParameterSetName = 'TeamNameElements')]
+        [Parameter(ParameterSetName = 'TeamSlugElements')]
+        [string] $OwnerName,
+
+        [Parameter(ParameterSetName = 'TeamNameElements')]
+        [Parameter(ParameterSetName = 'TeamSlugElements')]
+        [string] $RepositoryName,
+
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName = 'TeamNameUri')]
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName = 'TeamSlugUri')]
+        [Alias('RepositoryUrl')]
+        [string] $Uri,
+
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName = 'TeamNameElements')]
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName = 'TeamSlugElements')]
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName = 'TeamNameUri')]
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName = 'TeamSlugUri')]
+        [ValidateNotNullOrEmpty()]
+        [string] $OrganizationName,
```

Comment:
Same feedback/question as above regarding whether we _actually_ need `OrganizationName` to be passed-in here, or whether we can just re-use `OwnerName` as the `OrganizationName`...  I'm genuinely not sure, but I don't _think_ you can apply org teams to a non-org repo.